src/components/ui/InteractWithBurner.tsx

```diff
@@ -0,0 +1,176 @@
+)
+
+const infoSealWrapper = classnames(
+  displayFrom('md'),
+  margin('mr-4'),
+  dropShadow('drop-shadow-info-seal')
+)
+
+const ActionsButtons = ({
+  tertiaryButton,
+  primaryButton,
+}: {
+  tertiaryButton?: ButtonProps
+  primaryButton?: ButtonProps
+}) => {
+  return (
+    <div className={buttonsWrapper}>
+      {primaryButton?.text && (
+        <Button small type="primary" onClick={primaryButton.action}>
+          {primaryButton.text}
+        </Button>
+      )}
+      <Button
+        type="tertiary"
+        gradientFont
+        small
+        onClick={tertiaryButton?.action}
+      >
+        {tertiaryButton?.text}
+      </Button>
+    </div>
+  )
+}
+
+export default function ({
+  showAttention,
+  mainText,
+  headerText,
+  primaryButton,
+  tertiaryButton,
+  sadSeal,
+}: {
+  mainText: string
+  headerText: string
+  tertiaryButton?: ButtonProps
+  primaryButton?: ButtonProps
+  sadSeal?: boolean
+  showAttention?: boolean
+}) {
+  const [show, setShow] = useState(false)
+
+  function renderButtons() {
+    return (
+      <ActionsButtons
+        primaryButton={primaryButton}
+        tertiaryButton={tertiaryButton}
+      />
+    )
+  }
+
+  return (
+    <div className={contentWrapper}>
+      <div className={backgroundWrapper}>
+        <div className={infoSealWrapper}>
+          <InfoSeal sadSeal={sadSeal} />
+        </div>
+        <div className={positionWrapper}>
+          <div className={classnames(positionWrapper, gap('gap-y-1'))}>
+            <div className={headerTextWrapper(show)}>
+              <span className={classnames(display('flex'), gap('gap-x-1'))}>
+                <div
+                  className={classnames(
+                    displayTo('md'),
+                    dropShadow('drop-shadow-info-seal')
+                  )}
+                >
+                  <SmallInfoSeal />
+                </div>
+                <AccentText color="text-formal-accent" large bold>
+                  {showAttention && (
+                    <span className={textColor('text-secondary')}>
+                      Attention:{' '}
+                    </span>
+                  )}
+                  {headerText}
+                </AccentText>
+              </span>
+              <Button
+                center
+                type="tertiary"
+                onClick={() => {
+                  setShow(!show)
+                }}
+              >
+                <div className={classnames(width('w-4'), margin('mt-2'))}>
+                  <span className={displayTo('md')}>
+                    <Arrow pulseDisabled open={show} />
+                  </span>
+                  <span className={displayFrom('md')}>
+                    <Arrow pulseDisabled open={!show} />
+                  </span>
+                </div>
+              </Button>
+            </div>
+            {show && <StatusText>{mainText}</StatusText>}
+          </div>
+          <div className={displayFrom('md')}>{renderButtons()}</div>
```

Comment:
You don't need this function, simply do:
```
<ActionsButtons
        primaryButton={primaryButton}
        tertiaryButton={tertiaryButton}
      />
```
Or use a `const buttons = <ActionsButtons
        primaryButton={primaryButton}
        tertiaryButton={tertiaryButton}
      />`

Reply:
Constant should work as well as the function, but it's better practice I guess

Reply:
fixed, ty